celestia/peers.txt

```diff
@@ -1,17 +1,17 @@
-4ebe56795fc5da1c0dc5f7acd5760d5c8fd4d250@5.9.87.216:33357
-bb72dcd76b96214c4e647a1ebfe47c9e541d5325@185.182.186.202:26656
-be5ea8954fa452ad464ae8a7067f01bf5a0da556@147.135.144.53:26656
-95c00bb9f24e907891e9a98c4ad506af093a7944@87.106.237.243:26656
-4ad28c8a0b04bac4aac441b105c815c449e2182f@65.108.61.189:26656
-c7dac72f630d2b3c3354ed664bcd6704a8f9e627@51.79.229.177:26656
-ce2873ef3893f0da50eb27892761e9c06e704944@159.203.143.94:26656
-1b92a439555a19ee6edea85b184abb6873757dae@51.89.173.96:26656
-ea9d858371ba86ae8f9c5b292c105329029c9efd@5.199.174.19:26656
-5bc86308e2b3730ce611abf622c8be752780f934@5.199.172.53:26656
-acca7837e4eb5f9dc7f5a94ed1d82edda6931ff8@rpc-celestia.pops.one:26656
-f5bce79feec9068154e7f50fe8db7a8191a58663@82.220.38.204:10056
-629c36a551ee1a3c8bb3ecee6e00a9977a60e53e@65.108.46.248:56656
-d0c530290267e1c539f27061c9446d5042eaf1ca@161.97.180.20:26656
-a41b4bc451b1b71d537aa1018226e08d7fa7e44e@5.255.77.44:26656
-2f05b07cde80c5b08ee0159658f7977c7765f966@148.113.17.55:26756
-5001de72be39622c9dc34f2117eccc3f3fca8a7a@celestia-mainnet-01.twinstake.io:26756
+-4ebe56795fc5da1c0dc5f7acd5760d5c8fd4d250@5.9.87.216:33357 
+-bb72dcd76b96214c4e647a1ebfe47c9e541d5325@185.182.186.202:26656 
+-be5ea8954fa452ad464ae8a7067f01bf5a0da556@147.135.144.53:26656 
+-95c00bb9f24e907891e9a98c4ad506af093a7944@87.106.237.243:26656 
+-4ad28c8a0b04bac4aac441b105c815c449e2182f@65.108.61.189:26656 
+-c7dac72f630d2b3c3354ed664bcd6704a8f9e627@51.79.229.177:26656 
+-ce2873ef3893f0da50eb27892761e9c06e704944@159.203.143.94:26656 
+-1b92a439555a19ee6edea85b184abb6873757dae@51.89.173.96:26656 
+-ea9d858371ba86ae8f9c5b292c105329029c9efd@5.199.174.19:26656 
+-5bc86308e2b3730ce611abf622c8be752780f934@5.199.172.53:26656 
+-acca7837e4eb5f9dc7f5a94ed1d82edda6931ff8@rpc-celestia.pops.one:26656 
+-f5bce79feec9068154e7f50fe8db7a8191a58663@82.220.38.204:10056 
+-629c36a551ee1a3c8bb3ecee6e00a9977a60e53e@65.108.46.248:56656 
+-d0c530290267e1c539f27061c9446d5042eaf1ca@161.97.180.20:26656 
+-a41b4bc451b1b71d537aa1018226e08d7fa7e44e@5.255.77.44:26656 
+-2f05b07cde80c5b08ee0159658f7977c7765f966@148.113.17.55:26756 
+-5001de72be39622c9dc34f2117eccc3f3fca8a7a@celestia-mainnet-01.twinstake.io:26756
```

Comment:
@bo-everstake to clarify we need to make sure that we're not modifying this file. thanks!